qiskit/pulse/pulse_lib/continuous.py

```diff
@@ -184,7 +184,7 @@ def gaussian_deriv(times: np.ndarray, amp: complex, center: float, sigma: float,
         ret_gaussian: Return gaussian with which derivative was taken with.
     """
     gauss, x = gaussian(times, amp=amp, center=center, sigma=sigma, ret_x=True)
-    gauss_deriv = -x/sigma*gauss
+    gauss_deriv = -x / sigma * gauss
```

Comment:
Shouldn't this be -(x-center)/sigma * gauss? if gauss = amp exp[-(x-center)^2/2 sigma^2]

Reply:
`x` here is `x = (times - center) / sigma` (line 163), unfortunately. Good eye, though!

Reply:
So in the end we get a coefficient of `- (x - center) / sigma ** 2` :)

Reply:
Ok great!